include/Core/UI/Component/DashboardWidget.php

```diff
@@ -24,11 +24,18 @@ public function load() {
 	 * Adds the dashboard widget to display Email Log activity.
 	 */
 	public function register() {
-		wp_add_dashboard_widget(
-			'email_log_dashboard_widget',
-			__( 'Email Logs Summary', 'email-log' ),
-			array( $this, 'render' )
-		);
+		$dashboard_status = false;
+		$options          = get_option( 'email-log-core' );
+		if( isset($options['dashboard_widget_status']) )
```

Comment:
Move this condition here https://github.com/sudar/email-log/blob/dev/2.3.0/include/Core/UI/UILoader.php#L58

Also make sure every if statement has `{` even if it is just one statement.